src/components/DiveLog/DiveLog.js

```diff
@@ -25,7 +25,7 @@ export default function DiveLog({
 
   return (
     <>
-      <DiveLogDetailsCard onClick={handleCardToggle}>
+      <DiveLogDetailsCard>
         <DiveLogCard onClick={() => handleCardToggle()}>
```

Comment:
Why do you create an arrow function if you are not trying to hand down any value? :)